### ISSUE TITLE
add size units

### ISSUE THREAD

**Issue by AndreiKingsley** [AndreiKingsley/lib-ggdsl#32]
Add size unit types (.px/.dp) for size aesthetics.

**Comment by devcrocod** [comment]
also add percentages

**Comment by devcrocod** [comment]
@AndreiKingsley What's the state of this issue? Are we including it in the next release?

**Comment by AndreiKingsley** [comment]
No, there're several problems with Lets-Plot size units, but we still need this enhancement.

**Comment by devcrocod** [comment]
Okay, Is there a corresponding issue in lets-plot that we can track?

**Comment by AndreiKingsley** [comment]
No, it's not and lets-plot issue, lets-plot just has its own units.

**Comment by AndreiKingsley** [comment]
Size units in Lets-plot work like this:
1) `size` and `stroke` aesthetics (line width, points size, border line width and others) are converted as follows into pixels https://github.com/JetBrains/lets-plot/blob/master/plot-base/src/commonMain/kotlin/org/jetbrains/letsPlot/core/plot/base/aes/AesScaling.kt
Here you can see that it is just multiplication by coefficients, so you can make pixels in kandy API and just multiply by the opposite coeff one inside the translation - nothing complicated.
2) Text has its own size (It goes directly into the SVG element attribute, and how it will be rendered further does not depends on Lets-Plot).
3) Width, height (aesthetics and parameters) have relative units of measurement - a fraction of the "resolution" on the corresponding axis.

So here we can do next: 
1) Add pixel units for common sizes
2) Add special units for width/height 
3) With text - don't know

Anyway we should add documentations for size.

**Comment by AndreiKingsley** [comment]
https://nbviewer.org/github/JetBrains/lets-plot-kotlin/blob/master/docs/examples/jupyter-notebooks/f-4.7.1/param_size_unit.ipynb
